File: total_amount_of_points/TotalPoints.py
def points(games):
    points = 0
    for i in games:
        resultString = i.split(":")
        we = int(resultString[0])
        they = int(resultString[1])
        if we > they:
            points += 3
        elif we == they:
            points += 1
    return points
print(points(["3:1", "3:3", "1:3"]))

File: total_amount_of_points/TotalPoints.js
function points(games) {
    let points = 0;
    for ( s of games) {
        let resultString = s.split(":");
        let we = parseInt(resultString[0]);
        let they = parseInt(resultString[1]);
        if (we > they) points += 3;
        else if ( we === they) points += 1;
    }
    return points
}
console.log(points(["3:1", "3:3", "1:3"]));
